package.json

```
@@ -48,7 +48,10 @@
     "license": "Apache-2.0",
     "overrides": {
         "react-refresh": "0.11.0",
-        "react-is": "17.0.2"
+        "react-is": "17.0.2",
+        "react-windowed-select": {
+            "@types/react": "^17.0.67"
```

Shouldn't this be a fixed version?

```suggestion
            "@types/react": "17.0.67"
```